lib/core/auth.js

```diff
@@ -260,7 +233,7 @@ export default class Auth {
     return this.$storage.getState('busy')
   }
 
-  request (endpoint, defaults, withResponse) {
+  request (endpoint, defaults) {
```

Comment:
Can we also move `request` to `RequestHandler` and initialize in per scheme instance? then we don't need `requestWith` anymore :)